src/styles.py

```diff
@@ -30,6 +30,10 @@ def _generate_filename(name: str) -> str:
     return f"{name.lower().replace(' ', '-')}.mplstyle"
 
 
+def _is_style_bright(params: RcParams):
+    return utilities.get_luminance(params["figure.facecolor"]) < 150
```

Comment:
```suggestion
    return utilities.get_luminance(params["axes.facecolor"]) < 150
```

Should check for the axes background, not the figure background, as that is where the edit pencil is located.